### ISSUE TITLE
Unzip GmailAttachment not working

### ISSUE THREAD

**Issue by milokmet** [tanaikech/UnzipGs#2]
Hi, can't get this working. 

I'd like to unzip password protected GmailAttachments, but I'm always getting an error:
`Error: Please set a file blob of zip file.`
The attachment.toString() returns GmailAttachment and not Blob as expected on L44: `if (blob_.toString() !== "Blob") {`

So I tried workaround to save the attachment to the DriveApp temp folder and tried to use it as in README
```
  var file = saveToTmp(data);
  var tmpFile = DriveApp.getFileById(file.getId());
  var res = UnzipGs.unzip(tmpFile.getBlob(), { password: "my pwd"});
```
But I got this error: `TypeError: e.subarray is not a function  at $(Code:40:67)  at t.Y(Code:40:309) `

Can you point me what I am doing wrong? Thank you

**Comment by djcedr** [comment]
Hi, have you tried unchecking v8 in the script settings? I initially got this error - after unchecking, I got another error, but not this one anymore -- see issue #3

**Comment by pol-rivero** [comment]
@milokmet Did you ever find a way to solve this? I'm encountering the same problem

**Comment by milokmet** [comment]
@p-rivero nope. This was only one of the reasons, i sticked to python on my server.

**Comment by pol-rivero** [comment]
@milokmet I see, guess I'll also have to find another way.
Thank you for letting me know

**Comment by mehdinathani** [comment]
did you find the solution?

**Comment by pol-rivero** [comment]
@mehdinathani Unfortunately no, I also opted for python and created a cron job to run hourly on my computer.
This means that incoming emails are not processed until I turn on my pc in the morning, but that's perfectly acceptable for my use case.
As an added bonus, the decryption password now stays safely on my computer.

**Comment by Hawiie13** [comment]
@tanaikech  I tried the [unzipminjs ](https://github.com/MadfishDT/unzipt/tree/master) here, its seems working in the V8 engine if you have time to look at it?



I just tweak the unzipminjs in the code in order to be used in another file.

<img width="631" height="290" alt="Image" src="https://github.com/user-attachments/assets/7739e372-e1fd-4469-a861-6d4e9028af5d" />
